src/components/cylc/GraphSubgraph.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <g class="c-graph-subgraph">
+    <rect
+      :width="subgraph.width"
+      :height="subgraph.height"
+      :x="subgraph.x"
+      :y="subgraph.y"
+      rx="50"
+      ry="50"
+      fill="none"
+      stroke-width="8px"
+      stroke="grey"
+      stroke-dasharray="50 50"
+      />
+    <text
+      :x="labelXPosition"
+      :y="labelYPosition"
+      font-family="Roboto"
+      alignment-baseline="middle" text-anchor="middle"
+      font-size="70px"
+      fill="black"
+      stroke-width=1.5
```

Comment:
Currently the white stroke outline is being drawn over the top of the text which makes it look thin and limits how wide we can make the stroke (mb).

I found a nicer way of doing this using "paint-order" to make the browser draw the stroke first, then draw the text over it.

```suggestion
      stroke-width=5
      paint-order="stroke";
```

This gives a generous margin to arrows:

![Screenshot from 2024-05-14 10-13-52](https://github.com/cylc/cylc-ui/assets/16705946/f705aa1a-6ea4-4a08-8dd1-bcb0e7d322c4)